src/operations/extrusions/slice/calculatePlane.js

```diff
@@ -0,0 +1,35 @@
+const { plane, vec3 } = require('../../../math')
+
+const toEdges = require('./toEdges')
+
+/**
+ * Calculate the plane of the given slice.
+ * NOTE: The points are assumed to be planar from the beginning.
+ * @param {slice} slice - the slice
+ * @returns {plane} the plane of the slice
+ * @example
+ * let myplane = toPlane(slice)
+ */
+const calculatePlane = (slice) => {
+  const edges = toEdges(slice)
```

Comment:
slices already have edges right ? why the need to recompute ?

Reply:
slice.calculatePlane() is a special function, and only used in slice.toPolygons(), which does a boolean intersect to convert edges to polygons.

normally, there's no need to know the plane of a slice.